Naugrimm/opentrans
# Bring package up to date

- Drop Support for PHP 7
- Move from Doctrine annotations to PHP Attributes
- Use the latest version of `naugrim/bmecat`
- Make PHPStan happy
- Remove all simple getters/setters and replace them by a magic `__call` method. a PHPStan extension is provided that detects all valid getters/setters for serializable Attributes.
- Add some example opentrans documents from various vendors and deserialize them in the tests. They are serialized again and then must match the original document
- Fixing many elements that were not correctly implemented. See the `CHANGELOG.md`
